_components/footers.md

```diff
@@ -6,16 +6,244 @@ title: Footers
 
 <div class="preview">
   <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Footer_FullUI_v1-930width.png">
+
+  <h3>Footer Big</h3>
+
+  <footer class="usa-footer usa-footer-big usa-sans" role="contentinfo">
+    <div class="grid-box usa-footer-return-to-top">
+      <a href="#usa-top-of-page">Return to top</a>
+    </div>
+    <div class="usa-footer-primary-section">
+      <div class="grid-box">
+        <nav class="usa-footer-nav width-two-thirds">
+          <ul class="usa-unstyled-list width-one-fourth">
+            <h3 class="usa-footer-primary-link">Topic</h3>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+          </ul>
+          <ul class="usa-unstyled-list width-one-fourth">
+            <h3 class="usa-footer-primary-link">Topic</h3>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+          </ul>
+          <ul class="usa-unstyled-list width-one-fourth">
+            <h3 class="usa-footer-primary-link">Topic</h3>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+          </ul>
+          <ul class="usa-unstyled-list width-one-fourth">
+            <h3 class="usa-footer-primary-link">Topic</h3>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+          </ul>
+        </nav>
+
+        <div class="usa-sign_up-block width-one-third">
+          <h3 class="usa-sign_up-header">Sign up</h3>
+
+          <label class="" for="email" id="">Your email address</label>
+          <input id="email" name="email" type="input-type-email">
+
+          <button type="submit">Sign up</button>
+        </div>
+      </div>
+    </div>
+
+    <div class="usa-footer-secondary-section usa-footer-big-secondary-section">
```

Comment:
For this class name, is each part of it a separate part? So is this a footer that's a secondary footer that's a section of a footer? Or is it a footer that's a secondary section of a footer? If the latter, I think `usa-footer-secondary_section` makes more sense.